frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -3973,4 +3983,92 @@ def zeros(self, resulting_type: ConversionType) -> Conversion:
             original_bit_width=1,
         )
 
+    def get_partition_name(self, partition: tfhers.TFHERSParams) -> str:
```

Comment:
I'm wondering why you generates names, I fell it should be the responsability of the user to do it.

Reply:
names are used to identify partitions, which is abstracted to the user. So generating unique names for a given set of parameters is how to differentiate between these partitions and I don't see why this should be left to the user.